nomad/structs/config/vault.go

```diff
@@ -61,6 +61,9 @@ type VaultConfig struct {
 	// URL such as "http://vault.example.com"
 	Addr string `mapstructure:"address"`
 
+	// JWTBackendPath is the path used to access the JWT auth method.
+	JWTBackendPath string `mapstructure:"jwt_backend_path"`
```

Comment:
I'm open to suggestions on how to call this field. I followed the language from https://developer.hashicorp.com/vault/docs/auth/jwt#via-the-cli

> The default path for the JWT authentication backend is `/jwt`

Reply:
Maybe `JWTAuthBackendPath`? I don't feel strongly about it though.